lib/scenic/adapters/postgres.rb

```diff
@@ -70,16 +70,60 @@ def drop_materialized_view(name)
 
       # Refreshes a materialized view from its SQL schema.
       #
-      # @param name The name of the materialized view to refresh..
+      # @param name The name of the materialized view to refresh.
       # @return [void]
       def refresh_materialized_view(name)
         execute "REFRESH MATERIALIZED VIEW #{name};"
       end
 
+      # Caches indexes on the provided object before executing the block and
+      # then reapplying the indexes. Errors in applying the indexes are caught
+      # and logged to output.
+      #
+      # This is used when updating a materialized view in order to maintain all
+      # applicable indexes after the update.
+      #
+      # @param on The name of the object we are reapplying indexes on.
+      # @return [void]
+      def reapplying_indexes(on: name, &block)
```

Comment:
What am I supposed to do here. I want to say this method requires a block. I also prefer `yield` to `block.call` so... should this just be ignored?